### PR TITLE
Add info on where api, account and version can be found

### PR DESCRIPTION
**Pull request by fakela**
This is in accordance to 21, 22 and 23 of this issue #242

### CODE REVIEW

File: apis-overview/endpoints/prediction.md [hunk 1]
```diff
@@ -18,12 +18,12 @@ To make a prediction on your API, you must select your API version **__<api_vers
 ```
 https://api.mindee.net/v1/products/<account_name>/<api_name>/<api_version>/predict
 ```
-- **account_name**: refers to the username or organization name of the account that created the API. For the off-the-shelf APIs, this will read as 'mindee'.
-- **api_name**: refers to the name of your document parsing API
+- **account_name**: refers to the username or organization name of the account that created the API. For the off-the-shelf APIs, this will read as 'mindee'. This can be found at the top right of the page.
```
Comment (fharper) [inline apis-overview/endpoints/prediction.md:21]: Which page?

**Review by fharper — CHANGES_REQUESTED** [review]

File: apis-overview/endpoints/prediction.md (hunk 1 above)
Comment (fakela) [reply apis-overview/endpoints/prediction.md:21]: screen should be a better way to describe it.

Comment (fharper) [reply apis-overview/endpoints/prediction.md:21]: Still, which screen? Unless I'm mistaken, when the user read this page, it's not about being in any specific page, it's information about using the prediction endpoint no?

Comment (fakela) [reply apis-overview/endpoints/prediction.md]: In the suggestion, you mentioned adding the information like I did on this page
https://developers.mindee.com/docs/endpoints
But this API is generated so I think that information is necessary.

**Review by fharper — APPROVED** [review]
LGTM

**Pull request merged**